### ISSUE TITLE
Preparing molecule regression dataset for spectral prediction

### ISSUE THREAD

**Issue by Rhys-McAlister** [aimat-lab/gcnn_keras#116]
Hi, I'm trying to follow along with the `workflow_molecule_regression.ipynb` notebook to use the dmpnn model to predict ir spectra that are represented by vectors 1801 elements long. 

I'm a little bit lost as to how to make this work. When doing so in pytorch I normally set the output dimension to be 1801 such that the loss function compares the test vectors against the output vectors but when modifying the model config the output appears to be 1369 elements long, which is the number of rows in the y_train dataset. 

Any help in how I can alter this notebook to achieve the same thing would be greatly appreciated.
